### PR TITLE
Set fee recipient as global env

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Set fee recipient as global env

### CODE REVIEW

**Comment by pablomendezroyo** [comment]
Depends on https://github.com/dappnode/DNP_DAPPMANAGER/pull/1404

**Comment by nflaig** [comment]
We are also setting the fee recipient on the beacon node, see https://github.com/dappnode/DAppNodePackage-Lodestar-Prater/pull/39 on why this is done.

I think we need to either apply the same logic to the beacon node or remove the `suggestedFeeRecipient` flag from beacon node as it is not strictly required, just a fallback.

**Comment by nflaig** [comment]
`FEE_RECIPIENT_ADDRESS` will always be overriden, meaning the fee recipient set in setup wizard is unused, shall this be removed as well?

https://github.com/dappnode/DAppNodePackage-Lodestar-Prater/blob/054f218ad6288ed79ed609100151c1eb57d32df4/setup-wizard.yml#L23-L33

File: beacon-chain/entrypoint.sh [hunk 1]
```diff
@@ -34,6 +34,14 @@ if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_P
     fi
 fi
 
+# Chek the env FEE_RECIPIENT_PRATER has a valid ethereum address if not set to the null address
```
Comment (nflaig) [inline beacon-chain/entrypoint.sh]:
```suggestion
# Check the env FEE_RECIPIENT_PRATER has a valid ethereum address if not set to the null address
```

File: validator/entrypoint.sh [hunk 2]
```diff
@@ -15,6 +15,14 @@ if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_P
     fi
 fi
 
+# Chek the env FEE_RECIPIENT_PRATER has a valid ethereum address if not set to the null address
```
Comment (nflaig) [inline validator/entrypoint.sh]:
```suggestion
# Check the env FEE_RECIPIENT_PRATER has a valid ethereum address if not set to the null address
```

**Comment by pablomendezroyo** [comment]
Nice cartch it should be removed also from `setup-wizard`

**Review by nflaig — APPROVED** [review]
lgtm, just minor typo which could be fixed

**Review by dsimog01 — APPROVED** [review]

**Comment by pablomendezroyo** [comment]
not needed

**Pull request closed without merging**